app/controllers/users_controller.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  before_action :logged_in_user, except: %i[new create]
+  before_action :load_user, except: %i[index new create]
+  before_action :correct_user, only: %i[edit update]
```

Comment:
```suggestion
  before_action :correct_user, only: %i(edit update)
```
Dùng ngoặc tròn cho %i,w... các thứ nha e. Khỏi bị nhầm mảng